lib/commands/web-view/list-web-views.ts

```diff
@@ -0,0 +1,31 @@
+///<reference path="../../.d.ts"/>
+"use strict";
+import helpers = require("../../common/helpers");
+
+export class ListWebViewsCommand implements ICommand {
+	constructor(private $webViewService: IWebViewService,
```

Comment:
In webViewService constructor we call ensureCordovaProject. This means that the list command cannot be executed outside of project. Is this the desired behavior (I assume yes, but I've decided to ask )